Source/CheckoutAPIClient.swift

```diff
@@ -289,4 +341,12 @@ public class CheckoutAPIClient {
         }
     }
 
+    private static func retrieveSDKAppID(from sdkAppIDProvider: SDKAppIDProviding) -> String {
+        switch sdkAppIDProvider.sdkAppID() {
+        case .success(let sdkAppID):
+            return sdkAppID.appId
+        case .failure(let error):
+            return error.stringValue
```

Comment:
I can't remember what we discussed, but should we be sending an error value as the SDK App ID in logs? Wouldn't it pollute the logs?

Although I guess there isn't another way to log the error scenario here, as we need the SDK App ID

Reply:
I don't think we ever discussed this. On 3DS I think we just use empty string

Reply:
Looks like on 3DS2 we return a new UUID and log the error, could we do that here? Or would it be better to have this story focus on a new UUID and then the exception as another?

![image](https://user-images.githubusercontent.com/59568833/115247761-fbc4c680-a11e-11eb-911d-577031ea4271.png)

Reply:
We can handle this at a later point in time. Right now I think we just want to get this initial log done and move onto other stuff.